src/mongo/db/storage/hse/src/hse_global_options.h

```diff
@@ -68,17 +64,15 @@ class KVDBGlobalOptions {
 
 private:
     static const int kDefaultForceLag;
-    static const std::string kDefaultCompressionStr;
-    static const std::string kDefaultCompressionMinBytesStr;
     static const bool kDefaultEnableMetrics;
+    static const std::string kDefaultCompressionDefaultStr;
```

Comment:
Two "Default"s in that variable name. Is that intentional?

Reply:
The rparam is "compression.default". That first default is the default value for the rparam. The name here is less than ideal, but I am just following the pattern